pkg/api/cloudslack/error.go

```diff
@@ -0,0 +1,35 @@
+package cloudslack
+
+type CloudSlackErrorType string
+
+const (
+	CloudSlackErrorBadRequest CloudSlackErrorType = "BadRequest"
```

Comment:
Can we rename this error? Bad request sounds a bit too generic to assume this is always "quota exceeded" case. So WDYT about renaming this to `QuotaExceeded`, both on client and server?